ross/rotor_assembly.py

```diff
@@ -3821,10 +3821,21 @@ def rotor_example_6dof():
     >>> import ross as rs
     >>> import numpy as np
     >>> rotor6 = rs.rotor_assembly.rotor_example_6dof()
-    >>> camp6 = rotor6.run_campbell(np.linspace(0,400,101),frequencies=18)
 
-    # plotting Campbell Diagram
+    Plotting rotor model
+    >>> fig = rotor6.plot_rotor()
+    >>> fig.show()
+
+    Running modal
+    >>> rotor_speed = 100.0 # rad/s
+    >>> modal6 = rotor6.run_modal(rotor_speed)
+    >>> print(f"Undamped natural frequencies:\n {modal6.wn}")
+    >>> print(f"Damped natural frequencies:\n {modal6.wd}")
```

Comment:
You also have to provide the output here for these prints, otherwise the test will fail.
Also, to avoid having to write big arrays in the docstrings you can use ellipsis to shorten these arrays.
Here is an example:
```python
    >>> bearing0.K(frequency) # doctest: +ELLIPSIS
    array([[[1000000., 1000000., ...
    >>> bearing0.C(frequency) # doctest: +ELLIPSIS
    array([[[200., 200., ...
```